### ISSUE TITLE
Live Job Event Log timeout

### ISSUE THREAD

**Issue by ernl** [jhuckaby/Cronicle#512]
## Summary
When running a job on Cronicle, the live log feed returns the following error: "Log Watcher: Server Connect Error: Error: timeout (http://10.0.0.241:3012)"; I am unable to see the live job output, it only becomes available after the job has been completed.

I'm running a python script using the Shell Plugin

## Steps to reproduce the problem

Shell code:
/home/eric/dev/environments/project/bin/python /home/eric/dev/project/test.py

## Your Setup
I am running Cronicle on a Ubuntu VPS (ARM architecture).

### Operating system and version?

Ubuntu 20.04.4 LTS

### Node.js version?

Node v16.16.0

### Cronicle software version?

Latest version, currently v0.99

### Are you using a multi-server setup, or just a single server?
Single server setup

### Can you reproduce the crash consistently?
Yes, it always happens whenever I run a job.

## Log Excerpts
Job log file: https://imgur.com/a/PCCD9T5
```
Log Watcher: Connecting to server: http://10.0.0.241:3012...

Log Watcher: Server Connect Error: Error: timeout (http://10.0.0.241:3012)
```

Thank you in advance,
ernl

**Comment by mikeTWC1984** [comment]
Yeah, live logs doesn't work well with things like that. If you just have 1 node setting below config might help:
custom_live_log_socket_url
Although it your case it might be blocked by browser (check console output).
The only good solution is to use centralized live log api (currently it requires direct connection to the node). It is expected in V2 of cronicle. In a mean time you can check my fork, I replaced websocket api with long polling (via master node) - [cronicle-edge](https://github.com/cronicle-edge/cronicle-edge)
